project_tier/exercises/migrations/0005_auto_20180424_1411.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.0.4 on 2018-04-24 18:11
```

Comment:
The Travis CI build failed with this migration, I'm not sure why.

>CommandError: Conflicting migrations detected; multiple leaf nodes in the migration graph: (0005_auto_20180424_1411, 0006_auto_20180424_1252 in exercises).

Full report: https://travis-ci.org/ProjectTIER/projecttier.org/builds/376945972?utm_source=github_status&utm_medium=notification